### ISSUE TITLE
Handle GTK3's drag-to-move menu bar better

### ISSUE THREAD

**Issue by jorams** [stumpwm/stumpwm#180]
So, apparently GTK3 repurposes the menu bar as a title bar, so that when you click an empty part of it and move the cursor, it moves the window. StumpWM doesn't handle this nicely.

A simple test case:
1. (v)Split a frame into three.
2. Open Emacs in the middle one
(2.5 Enable the emacs menu bar)
3. Click on an empty part of the menu bar and move the cursor to the right.

This causes the window to move to the last frame of the three.

There are two issues here:
1. StumpWM interprets `:configure-request` repositioning coordinates as relative to the parent. I don't know why that is, but it causes the new coordinates to be roughly double the previous ones.
2. Changing the coordinate relationship to `:root` "fixes" that problem, but causes the window to move to the left instead when moving the cursor to the left. This is because to find the frame to place the moved window in, `FIND-FRAME` uses `>=` and `<=`. The new coordinates are one pixel to the left, so they fall in the frame to the left.

Number 1 is easily fixed (caveat being 2), but I don't know if that breaks other things.
Number 2 is slightly more complicated. A quick hack is to add a small amount (1 is not always enough) to the coordinates we search with, causing such small offsets to be ignored.

Perhaps there's also a way to disable this behaviour from GTK3?

**Comment by dbjergaard** [comment]
I just tried this (with emacs).  I can't get anything really crazy to happen, though I suspect my emacs may have been built with gtk2.  I tried with firefox and evince and while I can't get anything super weird to happen, every once and a while the evince window will move randomly to other frames depending on mouse movement.  

It would be nice to be able to drag windows across frames.  If we could support that, this would just become a special case.
